docs/howtoguides/how_to_not_fix_related_usns.md

```diff
@@ -0,0 +1,65 @@
+# How to not fix related USNs
```

Comment:
@s-makin feel free to jump in here.

I think we should have an overall "how to use `pro fix`" how-to-guide. Then this new doc and the existing "how to use --dry-run" can both be sections in the overall fix how-to-guide.

That's a bit more work and we can treat that as a follow-up PR if you agree and prefer

Reply:
I definitely agree that there's a lot more that can be said about `pro fix` and the situations in which it can be useful. 

From the conversation we had earlier in the week (or last week?) about the request for specific pages from the Support team, I think we need to be careful about making sure we have discoverability for them (so they don't have to hunt through long pages for the specific bits they're looking for) vs. helping users to understand the functionality better. 

So, we *could* have a tutorial on the different functions of `pro fix` and how it can be used in different situations (using a VM sandbox) to teach users about it, and then having very specific how-to guide(s) about achieving specific tasks. Having a single how-to guide with all the different aspects of `pro fix` may not be desirable, but what we can do if we have a few `pro fix` how-to guides is have a section in the how-to section specifically about `pro-fix` and keep the guides "modular" under that heading. Let's have a follow-up meeting about it and figure out exactly what content we want/need around `pro fix`, and that will make it much easier to figure out what should go where and which pages can be combined/separate.

But either way I think it would make sense to do all that as a follow-up PR rather than holding this one up. It makes more sense to land this as a separate how-to now and have the option to reorganise it later if we need to.

(Sorry for the wall of text lol)

Reply:
I agree, let discuss about the organization of the docs and I can come up with a follow up PR once we have decided the best path forward